src/main.ts

```diff
@@ -1,4 +1,9 @@
-export { default as createProcessor, skip } from "./process.js";
-export { pinToLocal, pinToProvider } from "./process/pinContentToIpfs.js";
-export { default as truncate } from "./process/truncate.js";
-export { default as symetricEncrypt } from "./process/symetricEncrypt.js";
+export { default as createProcessor, skip, RowObject } from "./processor.js";
+export {
+  pinToLocal,
+  pinToProvider,
+  Pinner,
+  type FileContent,
+} from "./processors/pinContentToIpfs.js";
+export { default as truncate } from "./processors/truncate.js";
```

Comment:
renamed process to processors for consistency in avoiding potential collisions with the reserved word.

Reply:
Whoops, thought I already did it. Ty sir.